tm/src/main/java/io/seata/tm/api/DefaultGlobalTransaction.java

```diff
@@ -120,7 +120,7 @@ public void commit() throws TransactionException {
             return;
         }
         assertXIDNotNull();
-        int retry = COMMIT_RETRY_COUNT;
+        int retry = COMMIT_RETRY_COUNT <= 0 ? 1 : COMMIT_RETRY_COUNT;;
```

Comment:
Is it better to use the default value when <=0?

Reply:
fixed.